MapboxNavigation/NavigationMapView.swift

```diff
@@ -519,6 +521,16 @@ open class NavigationMapView: MGLMapView {
             style.removeLayer(arrowCasingSymbolLayer)
         }
     }
+    
+    func mutateBaseCameraStops(multiplier: Double) -> [Int: MGLStyleValue<NSNumber>] {
```

Comment:
This method doesn’t do anything to `multipler`; it uses `multiplier` to return a new value without any side effects. So rename this method to describe what it returns, avoiding any verbs.